frontends/api/src/test-utils/factories/learningResources.ts

```diff
@@ -129,6 +129,13 @@ const learningResourceOfferor: Factory<LearningResourceOfferor> = (
     code: faker.helpers.unique(faker.lorem.words),
     name: faker.helpers.unique(faker.lorem.words),
     channel_url: faker.internet.url(),
+    offerings: repeat(faker.lorem.word),
+    audience: repeat(faker.lorem.word),
+    formats: repeat(faker.lorem.word),
+    fee: repeat(faker.lorem.word),
+    certifications: repeat(faker.lorem.word),
+    content_types: repeat(faker.lorem.word),
+    more_information: repeat(faker.internet.url),
```

Comment:
This should just be `faker.internet.url` now, I think.

The reason that Typescript isn't giving you an error here is that it's declared as `LearningResourceOfferor`. We should change that to `LearningResourceOfferorDetail`. (We could also make two separate factories. But as long as `LearningResourceOfferorDetail` is a superset of `LearningResourceOfferor`, it's probably fine. Technically some objects will have too many attributes, but TS won't let you access them unless it's a Detail object.